scripts/src/test.sh

```diff
@@ -27,7 +27,7 @@ EOF
 }
 
 workspace_test() {
-  cargo nextest run --workspace "$@"
+  cargo nextest run --workspace "$@" --profile ci
```

Comment:
What does this profile mean?

Reply:
This is just profile for nextest configured in `.config/nextest.toml`. Since we're gonna run it mostly on CI it's named so.

Reply:
> 

But we run it manually also. With each `make pre-commit` call etc